src/Molecules/ActionCard/ActionCard.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+
+import { Trash } from 'styled-icons/boxicons-solid/Trash'
+import { Check } from 'styled-icons/material/Check'
+import { Edit } from 'styled-icons/material/Edit'
+
+import Card from '../../Atoms/Card'
+import Trans from '../../Atoms/Trans'
+import FlexBox from '../../Templates/FlexBox'
+
+let originalValue
```

Comment:
What is this for ?

Reply:
As the Editable components need to be updated to be rerendered, their value is modified using onChange listener.
I use `originalValue` to store Editable components value when clicking on Edit and set it back if we click on Cancel